src/main/java/org/prgrms/springbootbasic/controller/VoucherController.java

```diff
@@ -0,0 +1,67 @@
+package org.prgrms.springbootbasic.controller;
+
+import org.prgrms.springbootbasic.Menu;
+import org.prgrms.springbootbasic.VoucherType;
+import org.prgrms.springbootbasic.service.VoucherService;
+import org.prgrms.springbootbasic.view.ConsoleView;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class VoucherController {
+
+    private static final Logger logger = LoggerFactory.getLogger(VoucherController.class);
+    private final VoucherService voucherService;
+    private final ConsoleView consoleView;
+
+    public VoucherController(VoucherService voucherService, ConsoleView consoleView) {
+        this.voucherService = voucherService;
+        this.consoleView = consoleView;
+    }
+
+    public void run() {
+        while (process())
+            ;
+    }
+
+    private boolean process() {
+
+        logger.info("VoucherController.process() called");
```

Comment:
```suggestion
    private boolean process() {
        logger.info("VoucherController.process() called");
```
        이러한 개행하나하나가 가독성을 다르게 만들수 있습니다 :)